test/ducks/apps/__snapshots__/containers.spec.js.snap

```diff
@@ -74,28 +32,6 @@ exports[`Apps Containers (connected components): MyApplications should be render
       "subscribe": [Function],
     }
   }
-  storeSubscription={
-    Subscription {
-      "listeners": Object {
-        "clear": [Function],
-        "get": [Function],
-        "notify": [Function],
-        "subscribe": [Function],
-      },
-      "onStateChange": [Function],
-      "parentSub": undefined,
-      "store": Object {
-        "clearActions": [Function],
-        "dispatch": [Function],
-        "getActions": [Function],
-        "getState": [Function],
-        "replaceReducer": [Function],
-        "subscribe": [Function],
-      },
-      "unsubscribe": [Function],
-    }
-  }
-  uninstallApp={[Function]}
-  updateApp={[Function]}
+  t={undefined}
```

Comment:
Is it still interesting to keep the snapshots here? Since it's just these three global variables.

Reply:
Not really intersting, but maybe it allows to detect unpredicted regressions ?